includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -810,9 +810,11 @@ function_exists( 'wcs_order_contains_subscription' )
 	 * @return array|null     Array if processed, null if the new process is not supported.
 	 */
 	public function new_process_payment( WC_Order $order ) {
+		$manual_capture = $this->get_capture_type() === Payment_Capture_Type::MANUAL();
```

Comment:
Nitpick: We don't have to define a new variable. We could simply copy this statement inside brackets on line 817.

Reply:
Answered here: https://github.com/Automattic/woocommerce-payments/pull/7332#discussion_r1342853886